### PR TITLE
[SEMI-MODULAR] Psychologists now spawn with a PDA cartridge that allows them to use the send-all feature 

### PR DESCRIPTION
**Pull request by DuffCreeper**
## About The Pull Request

Adds a file into the modular items folder that basically gives the Psychologist their own version of the Medical cart that removes the bot access (since they're a psych) and adds instead the Send All feature. They are also given their own PDA which just allows them to spawn with the cart in their PDA roundstart.
This also though modifies their job file for the PDA to spawn hence funny semi-modular tag

## How This Contributes To The Skyrat Roleplay Experience

Allows Psychologists to reliably get the fact they exist out there without having to compete with everyone else yelling in general comms to notice them and instead, use PDAs which kinda grabs attention easily.
This will hopefully allow psychologists to more or less set up more appointments than just sitting on their ass, walking around the station hoping for something or continuously speaking into comms hoping someone hears what they said ontop of everything people are saying

## Changelog

:cl:
qol: Psychologists now spawn with a medical cartridge that allows them to use the PDA's 'Send All' feature
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/modular_items/code/psych_cart.dm [hunk 1]
```diff
@@ -0,0 +1,12 @@
+/obj/item/cartridge/psychologist
+	name = "\improper P.A.T cartridge"
+	desc = "Flaunt that Psychology Degree with this Cartridge!"
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/modular_items/code/psych_cart.dm]: Could be a bit more creative, but that's not necessarily a bad description either.
```suggestion
	desc = "Flaunt that Psychology degree with this cartridge!"
```

File: modular_skyrat/modules/modular_items/code/psych_cart.dm [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+	icon_state = "cart-m"
+	access = CART_MEDICAL
+	spam_enabled = 1
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/modular_items/code/psych_cart.dm]: Gotta use dem defines boi
```suggestion
	spam_enabled = TRUE
```

File: modular_skyrat/modules/modular_items/code/psych_cart.dm [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+/obj/item/pda/psychologist
+	name = "Psychologist PDA"
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/modular_items/code/psych_cart.dm]: Are all PDAs capitalized in such a fashion? I got a feeling the job name shouldn't be capitalized here.

File: modular_skyrat/modules/modular_items/code/psych_cart.dm [hunk 4]
```diff
@@ -0,0 +1,12 @@
+/obj/item/cartridge/psychologist
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/modular_items/code/psych_cart.dm]: Was there any reason why this isn't a subtype (child) of the medical cartridge?

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
Good idea, just a few things to take a look at first.

File: modular_skyrat/modules/modular_items/code/psych_cart.dm (hunk 2 above)
Comment (DuffCreeper) [reply modular_skyrat/modules/modular_items/code/psych_cart.dm]: Ack I forgot that simple rule, not making PRs for so long made me forget that

File: modular_skyrat/modules/modular_items/code/psych_cart.dm (hunk 4 above)
Comment (DuffCreeper) [reply modular_skyrat/modules/modular_items/code/psych_cart.dm]: I couldn't remember if that'll force the robot calling feature onto the cartridge when a psych doesn't exactly need the ability to call medbots if they're going to be in their office

**Comment by SpaceVampire** [comment]
This really makes the lawyer feel less special. I liked how Psychs would have to try and do other things like writing up flyers and leaving them around the station, instead of just defaulting to doing that. If you really wanted to send out a message you could always ask to borrow the cart that allows it as well, which was some extra interaction.

**Comment by YakumoChen** [comment]
I'm inherently against more crew having the innate ability to send long, rambling messages to my PDA without stealing/borrowing most sought-after cartridge in the game.

**Comment by GoldenAlpharex** [comment]
After thinking some more about it, I'm not exactly sure how beneficial this is going to be, considering that's just more PDA spam and it makes it more common to have a Send All cartridge. I don't think it's that hard to get clients as a Psych as is, I mean, hell, I even played psych like 10 months ago and it wasn't honestly too hard to find people that needed an appointment.

**Comment by nikothedude** [comment]
Couldn't this be done upstream?

**Comment by axietheaxolotl** [comment]
Hi, I play Psych. This is a good PR. Busy work isn't interaction, having to bug a lawyer, or the HoP to get a send all cartridge is just annoying, and isn't fun to do. Nor is Copy and Pasting the same message and going down the PDA list. Also, people are going on about PDA spam? How is getting 3 messages in a shift PDA spam? If you want to reduce PDA spam, get rid of the ability to get SPAM cartridges in maints, this will contribute very little to PDA spam, if anything.

**Comment by axietheaxolotl** [comment]
> I liked how Psychs would have to try and do other things like writing up flyers and leaving them around the station, instead of just defaulting to doing that.

This doesn't stop people from doing that, I've seen psychs still do that, even with a SPAM cartridge. This just makes life easier.

**Comment by DuffCreeper** [comment]
> After thinking some more about it, I'm not exactly sure how beneficial this is going to be, considering that's just more PDA spam and it makes it more common to have a Send All cartridge. I don't think it's that hard to get clients as a Psych as is, I mean, hell, I even played psych like 10 months ago and it wasn't honestly too hard to find people that needed an appointment.

Only way to get this cartridge is to basically having to steal the Psych's PDA from them to remove the cart. As far as I'm aware I don't think there's a loot table that includes all cartridges. So at a time there'll only really be a single dingle PDA P.A.T

**Comment by tf-4** [comment]
Powercreep just ask a lawyer

**Comment by axietheaxolotl** [comment]
> Powercreep just ask a lawyer

image

**Comment by tf-4** [comment]
Please do not follow me around github or I will report you to the Internet police

**Comment by ghost** [comment]
> If you want to reduce PDA spam, get rid of the ability to get SPAM cartridges in maints, this will contribute very little to PDA spam, if anything.

Or re-add the Block feature >->

**Comment by Gandalf2k15** [comment]
Yeah, I'm gonna have to agree that this isn't really needed, PDA spam is annoying.

**Pull request closed without merging**